src/plugins/maps_dashboards/yarn.lock

```diff
@@ -0,0 +1,229 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
do we need this file? I think we should look at the format other plugins are also following for this file if at all it exists.

Reply:
Yes, looked at anomaly detection and report plugins, they all have it.